packages/lit-dev-content/samples/examples/context-basics/elements.ts

```diff
@@ -0,0 +1,51 @@
+import {html, LitElement} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+import {provide, consume, createContext} from '@lit-labs/context';
+
+import {providerStyles} from './styles.js';
+
+const contextKey = Symbol('contextKey');
+// The value type for the context.
```

Comment:
The word context is reaching semantic satiation here. context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context

How about we reword to give another angle on it.

```suggestion
// The values that will be provided and consumed in this demo will
// be strings.
```